### ISSUE TITLE
0.30.2 - `use_native_ot: never` doesn't stop `RQ|3220|00`s

### ISSUE THREAD

**Issue by zxdavb** [zxdavb/ramses_cc#84]
```yaml
  ramses_rf:
    disable_discovery: false
    enforce_known_list: false
    # use_native_ot: always  # will still RQ|3EF0|00
    # use_native_ot: never  # will still RQ|3220|00RQ|3220|00
 ```

The never should stop *all* `RQ|3220` packets, including `RQ|3220|00`.

**Comment by zxdavb** [comment]
Solution is:
```python
if self._gwy.config.use_native_ot != "never":
    self._add_discovery_cmd(Command.get_opentherm_data(self.id, "00"), 60)
if self._gwy.config.use_native_ot != "always":
    self._add_discovery_cmd(_mk_cmd(RQ, Code._3EF0, "00", self.id), 60)
```

**Comment by zxdavb** [comment]
related to #53

**Comment by zxdavb** [comment]
This has been fixed sometime before **0.31.7**.

This regex pair is for future reference:
```regex
" 18:.* 10:.* 3220 (... ....00)": "18:000730 10:000000 3220 \\1 "
```
